### PR TITLE
Remove .includes string polyfill

### LINKED ISSUE #1880: messed up polyfill: "undefined is not a function" :-(
A weird new one definitely caused by that polyfill.

`t.includes("smc-file-tabs")`:

```
TypeError: undefined is not a function
    at j.createElement.j.createElement.j.createElement.shouldCancelStart (https://cloud.sagemath.com/static/smc-4e037b639d72bdde1e0a.js?4e037b639d72bdde1e0a:158:24127)
    at HTMLUListElement.t.handleStart (https://cloud.sagemath.com/static/smc-4e037b639d72bdde1e0a.js?4e037b639d72bdde1e0a:217:29383)
    at HTMLUListElement.function.e._wrapper.e._wrapper (https://cloud.sagemath.com/static/lib-4e037b639d72bdde1e0a.js?4e037b639d72bdde1e0a:118:28349)
```

... and a bit different `t.includes("smc-project-tab-sorter")`:

```
TypeError: undefined is not a function
    at _.createElement.shouldCancelStart (https://cloud.sagemath.com/static/smc-4e037b639d72bdde1e0a.js?4e037b639d72bdde1e0a:191:25235)
    at HTMLUListElement.t.handleStart (https://cloud.sagemath.com/static/smc-4e037b639d72bdde1e0a.js?4e037b639d72bdde1e0a:217:29383)
    at HTMLUListElement.function.e._wrapper.e._wrapper (https://cloud.sagemath.com/static/lib-4e037b639d72bdde1e0a.js?4e037b639d72bdde1e0a:118:28349)
```

time: 5 min

### PR DESCRIPTION
**Pull request by johnjeng**
Fixes #1880

### CODE REVIEW

File: src/smc-webapp/project_page.cjsx [hunk 1]
```diff
@@ -507,7 +507,7 @@ exports.ProjectPage = ProjectPage = rclass ({name}) ->
                     distance             = {3 if not IS_MOBILE}
                     bsStyle              = "pills"
                     style                = {display:'flex', overflow:'hidden', flex: 1}
-                    shouldCancelStart    = {(e)=>e.target.getAttribute('class')?.includes('smc-file-tabs')}
+                    shouldCancelStart    = {(e)=>'smc-file-tabs' in e.target.getAttribute('class')?}
```
Comment (williamstein) [inline src/smc-webapp/project_page.cjsx]: This is seriously wrong, e.g.,

```
coffee> 0 in [0]?
false
```

Unfortunately, you have to do something more complicated with an if...

File: src/smc-webapp/projects_nav.cjsx [hunk 2]
```diff
@@ -185,7 +185,7 @@ FullProjectsNav = rclass
             lockAxis             = {'x'}
             lockToContainerEdges = {true}
             distance             = {3 if not isMobile.tablet()}
-            shouldCancelStart    = {(e)=>e.target.getAttribute('class')?.includes('smc-project-tab-sorter')}
+            shouldCancelStart    = {(e) => 'smc-project-tab-sorter' in e.target.getAttribute('class')?}
```
Comment (williamstein) [inline src/smc-webapp/projects_nav.cjsx]: also wrong

**Comment by johnjeng** [comment]
Oh, playing around with coffeescript,
```
coffee> "asdf" in "asdf asdf"
false
```

Closing this PR.

**Comment by williamstein** [comment]
in works like for python for lists, but not for strings.  In any case, this is probably not worth it; maybe we should just close 1880 too?

**Pull request closed without merging**